internal/policy/container/has_unique_tag.go

```diff
@@ -58,6 +60,7 @@ func (p *hasUniqueTagCheck) getDataToValidate(ctx context.Context, image string)
 	options := []crane.Option{
 		crane.WithContext(ctx),
 		crane.WithAuthFromKeychain(authn.PreflightKeychain(ctx, authn.WithDockerConfig(p.dockercfg))),
+		option.RetryOnceAfter(5 * time.Second),
```

Comment:
complete nit: a constant to make it configurable.

Reply:
I don't really want to expose this out to end users, then have something else to trouble-shoot.